src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs

```diff
@@ -126,7 +126,18 @@ public async Task<IEnumerable<AssessmentOrganisationSummary>> GetAssessmentOrgan
 
                 var assessmentOrganisationSummaries =
                     await connection.QueryAsync<AssessmentOrganisationSummary>(
-                        "select EndPointAssessorOrganisationId as Id, EndPointAssessorName as Name, EndPointAssessorUkprn as ukprn, OrganisationData from [Organisations]");
+                        "select EndPointAssessorOrganisationId as Id, " +
+                        "EndPointAssessorName as Name, " +
+                        "EndPointAssessorUkprn as ukprn, " +
+                        "OrganisationData, " +
+                        "OrganisationTypeId, " +
+                        "ot.Type as OrganisationType, " +
+                        "o.Status, " +
+                        "JSON_VALUE(OrganisationData,'$.Email') as Email " +
+                        "from [Organisations] o " +
+                        "join OrganisationType ot " +
+                        "on ot.id = o.OrganisationTypeId " +
+                        "and ot.Status = 'Live'");
```

Comment:
Just a check but is this similar to the behaviour of others? should we not return all and filter after on the OrganisationType status

Reply:
I can remove that, but it's making sure that if there are 2 org types, you only use the org type that is live. It's not the status of the org, but rather of the org type.